core/src/network/actions/custom_send.rs

```diff
@@ -37,11 +32,11 @@ pub async fn custom_send(
     let action_wrapper = ActionWrapper::new(Action::SendDirectMessage(direct_message_data));
     dispatch_action(context.action_channel(), action_wrapper);
 
-    async {
+    /* async {
```

Comment:
what happens here? how can this code just be commented out?

Reply:
I mentioned this in the PR text.  There's a followup to fix that.  Don't know how to do it yet.

Reply:
this got fixed